src/tools/diagnostic-messages.md

```diff
@@ -3076,18 +3255,19 @@ Add an initializer:
 const c = 'c';
 {% endprettify %}
 
-### const_set_element_type_implements_equals
```

Comment:
@bwilkerson It seems this one loses a destination since it was renamed to `const_set_element_not_primitive_equality` with added functionality. Is that expected or should `const_set_element_not_primitive_equality` have an anchor generated for this so Dart 2.19 users don't lose a link destination?

Reply:
Thanks! I'll hold off for your changes in https://dart-review.googlesource.com/c/sdk/+/290180 :)